cmd/pulumi-resource-docker-buildkit/main.go

```diff
@@ -275,11 +275,21 @@ func (k *dockerBuildkitProvider) dockerBuild(
 		platforms = append(platforms, v.StringValue())
 	}
 
+	cacheFrom := name
+	if !inputs["cacheFrom"].IsNull() {
+			index = inputs["cacheFrom"]
+	}
+
+	cacheTo := "type=inline"
+	if !inputs["cacheTo"].IsNull() {
+			index = inputs["cacheTo"]
+	}
+
 	args := []string{
 		"buildx", "build",
 		"--platform", strings.Join(platforms, ","),
-		"--cache-from", name,
-		"--cache-to", "type=inline",
+		"--cache-from", cacheFrom,
+		"--cache-to", cacheTo,
```

Comment:
I think you need to include these values in the output, too, to prevent spurious diffs (see e.g. 1dba92ed638c636e921ce465a8c7d29d2052bd8e).

Reply:
I added them to the outputs